src/layers/legacy/medVtkDataMeshBase/vtkMetaDataSetSequence.cxx

```diff
@@ -722,3 +722,26 @@ void vtkMetaDataSetSequence::ComputeTimesFromDuration()
         this->GetMetaDataSet(i)->SetTime((double) ( i )/ (double)(this->GetNumberOfMetaDataSets() ) );
     }
 }
+
+double* vtkMetaDataSetSequence::GetScalarRange(QString attribute)
+{
+    static double* val = new double[2];

```

Comment:
Why val is declared as static pointer ?

Reply:
I think, but maybe I'm wrong, it is not thread safe et we can have memory leak.
Some other solutions exists:
- std::array (with semantic move it's very clean and easy)
- std::vector  (with semantic move it's very clean and easy)
- use a static 'c' tab

Reply:
I agree, static is evil. I just copied this code from medInria 2, and needs to be improved.
I thought about changing the return type as you suggested, or pass the array by parameter.
It would however change the method signature and impact numerous other files, including some external projects.
Maybe in another PR? 
I guess removing the 'static' is already a good thing.

Reply:
Yes, remove the static it's the minimum to able to merge this PR :)
After, for other solution choose your favorite.